Competition2017/src/competition/subsystems/shooter_wheel/commands/UnjamShooterWheelCommand.java

```diff
@@ -0,0 +1,33 @@
+package competition.subsystems.shooter_wheel.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class UnjamShooterWheelCommand extends BaseShooterWheelCommand {
```

Comment:
Yeah, I'd be scared of a system which monitors wheel current and acts upon detected jamming... remember that the motor uses 20+ amps for 3+ seconds when spinning up the flywheel.